docs/tutorials/timeseries/forecasting-indepth.md

```diff
@@ -0,0 +1,248 @@
+
+**Local models** are simple statistical models that are specifically designed to capture patterns such as trend or seasonality.
+Despite their simplicity, these models often produce reasonable forecasts and serve as a strong baseline.
+Available local models include:
+
+- `ETS`
+- `ARIMA`
+
+If the dataset consists of multiple time series, we fit a separate local model to each time series — hence the name "local".
+This means, if we want to make a forecast for a new time series that wasn't part of the training set, all local models will be fit from scratch for the new time series.
+
+**Global models** are deep-learning-based algorithms that learn a single model from the entire training set consisting of multiple time series.
+AutoGluon relies on [GluonTS](https://ts.gluon.ai/stable/) for the implementation of global models in PyTorch and MXNet.
+Available global models include:
+
+- `DeepAR`
+- `SimpleFeedForward`
+- `Transformer`
+- `MQRNN`
+- `MQCNN`
+- `TemporalFusionTransformer`
+
+Finally, an **ensemble** model works by combining predictions of all other models.
+By default, `TimeSeriesPredictor` always fits a `WeightedEnsemble` on top of other models.
+This can be disabled by setting `enable_ensemble=False` when creating the predictor.
+
+For a list of tunable hyperparameters for each model, their default values, and other details see [Model zoo](#TODO).
+
+## How does AutoGluon evaluate performance of time series models?
+AutoGluon evaluates the performance of forecasting models by measuring how well their forecasts align with the actually observed time series.
+We can evaluate the performance of a trained predictor on `test_data` using the `evaluate` method
+```python
+# Fit a predictor to training data
+predictor = TimeSeriesPredictor(prediction_length=3, eval_metric="MAPE")
+predictor.fit(train_data=train_data)
+# Evaluate the predictor on test data
+predictor.evaluate(test_data)
+```
+For each time series in `test_data`, the predictor does the following:
+
+1. Hide the last `prediction_length` values of the time series.
+2. Generate a forecast for the hidden part of the time series using the model.
+3. Quantify how well the model's forecast matches the actually observed (hidden) values of the time series using the `eval_metric`.
+
+Finally, the scores are averaged over all time series in the dataset.
+
+The crucial detail here is that `evaluate` always computes the score on the last `prediction_length` time steps of each time series.
+The beginning of each time series (except the last `prediction_length` time steps) is only used to initialize the models before forecasting.
+
+### Multi-window backtesting
+
+If we want to perform _multi-window backtesting_ (i.e., evaluate performance on multiple forecast horizons generated from the same time series), we need to generate a new test set with multiple copies for each original time series.
+This can be done using a `MultiWindowSplitter`.
+```python
```

Comment:
Is it your intention to not run the code? Because that is what "```python" does. Refer to other tutorials for how they make the code run.

Reply:
This is intentional. The goal here is just to show how different things can be done in AGTS (without, e.g., training multiple predictors when rendering this document). Does that make sense, or do you think we should make some of these cells runnable?

Reply:
It is fine as is, thanks